pallenec

```diff
@@ -10,6 +10,12 @@ p:argument("source_file", "File to compile")
 p:flag("--emit-c", "Generate a .c file instead of an .so file")
 p:flag("--emit-asm", "Generate a .s file instead of an .so file")
 p:flag("--compile-c", "Compile a .c file generated by --emit-c")
+p:flag("--dump-ast", "Print ast from a .pln file")
```

Comment:
This could be an options instead of multiple flags. For instance, `--dump=ast`.

Reply:
At first I thought of doing with options, but when talking to Hugo about this he said to do it like this. I will read the documentation on options and make the change.

Reply:
Ask Hugo before changing to option, if he agrees then make the chage. Else keep like this.

Reply:
One nice thing about having  a single `--dump` flag is that it would make it clearer that it is only allowed to dump one thing at a time.

But there should be a way for an user running `pallenec --help` to find out what are the valid values that can be passed to `--dump`.

Reply:
It is possible to limit choices for the --dump option, but this is only possible on the latest version of argparse, meaning that I would have to bump the requirement on the rockspec to at version 0.7.0 or up. If it's ok to bump the requirement, I'll make the change.

Reply:
Upgrading the lib is not a problem